docs/dev/contributing.rst

```diff
@@ -185,22 +275,101 @@ You can install pre-commit by running:
 
   $ pip install pre-commit
 
+<<<<<<< HEAD
+It is best to install pre-commit in your existing enviromnment. After that simply install the necessary hooks with
+||||||| 6948ad29
+Then simply install the necessary hooks with
+=======
 Then simply install the necessary hooks with
+>>>>>>> main
 
 .. code-block:: console
 
   $ pre-commit install
 
-Now pre-commit will check all configured hooks against all the staged files and in many cases apply fixes for you before you commit.
+After doing this pre-commit will check all your staged files when commiting.
 
-When pre-commit applies fixes for you you will have to add them to the staging area again. The git staging area will never be modified by pre-commit.
+For example say that you've added the following new feature:
 
+<<<<<<< HEAD
```

Comment:
Well spotted. I think indeed black does not check rts files, and I missed this because I assumed it was picking up on the example I'd added as an actual merge conflict. Should be good now!